### PR TITLE
Introduce job for scanning new maven releases

### PR DESCRIPTION
**Pull request by fridex**

### CODE REVIEW

**Comment by fridex** [comment]
The MavenPopularAnalysis handler uses S3MavenIndex adapter to store maven index on S3. I think it should be utilized here as well. I will adjust implementation accordingly.

File: f8a_jobs/handlers/maven_releases.py [hunk 1]
```diff
@@ -0,0 +1,80 @@
...
+                              len(output))
+                for entry in output:
+                    self.run_selinon_flow('bayesianFlow', {
+                        'ecosystem': 'maven',
+                        'name': '{groupId}:{artifactId}'.format(**entry),
+                        'version': entry['version'],
+                        'recursive_limit': 0
+                    })
+            finally:
+                rmtree(java_temp_dir)
```
Comment (jpopelka) [inline f8a_jobs/handlers/maven_releases.py]: Could we use `f8a_worker.utils.tempdir()` ?

```python
from f8a_worker.utils import tempdir
with tempdir() as java_temp_dir:
```

**Comment by fridex** [comment]
> To me it'd make more sense to store the offset along the index itself, i.e. on S3 and not in postgres.

Reworked. I'm doing a test run, once finished I will squash commits. PTAL

**Comment by fridex** [comment]
Code works, but I'm kind of surprised - results are weird. I've requested to schedule analyses for newest 13 packages (maven index checker log: `Checking: 6186111 - 6186124`). Maven index checker returned only 3 entries:

[org.omnifaces:omnipersistence](https://mvnrepository.com/artifact/org.omnifaces/omnipersistence/0.3) in version 0.3
[org.omnifaces:omniutils ](https://mvnrepository.com/artifact/org.omnifaces/omniutils/0.5)in version 0.5
[org.omnifaces:optimusfaces](https://mvnrepository.com/artifact/org.omnifaces/optimusfaces/0.4) in version 0.4

All of them were released at the end of July. Index should be up2date (fresh run). Any ideas what could go wrong?

**Comment by jpopelka** [comment]
> (maven index checker log: Checking: 6186111 - 6186124). Maven index checker returned only 3 entries

I think it's Ok, we just [filter out](https://github.com/fabric8-analytics/maven-index-checker/blob/master/src/main/java/com/redhat/maven/index/checker/MavenIndexChecker.java#L384) artifacts with packaging other than 'jar' and duplicates.

**Comment by fridex** [comment]
Merging based on @jpopelka review. Thanks!

**Pull request merged**